### PR TITLE
Tweaked evaluation and worksheet template, Added Gnomesort Evaluations and Exercises, Added Splay Trees, Closes #10

### LINKED ISSUE #10: Splay Tree
With rotations, we should also be able to turn a binary search tree into a Splay tree

https://en.wikipedia.org/wiki/Splay_tree

### LINKED ISSUE #11: Huffman Coding
Can we put chits on cards, and do Huffman Encoding? 

https://en.wikipedia.org/wiki/Huffman_coding

The letter on the chit would have the frequency of the number on the Acorn card. 

How would we represent the intermediate nodes? Face-down cards like in skip-list?

### PR DESCRIPTION
**Pull request by qsebring3930**
-I noticed that even though worksheets worked on my local server, they did not work when pushed to the actual website. It was a minor typo in the permalink that I fixed
-Long evaluation questions wrapped strange and made the evaluation portion look messy so I added extra div containers
-Added splay trees algorithm, video, and example
-Added about 20 exercises and 5 evaluation questions for Gnomesort

### CODE REVIEW

File: _pages/splaytree.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ sidebar:
```
Comment (mgoadric) [inline _pages/splaytree.md:10]: Missing a learning goals sentence here.

File: _pages/splaytree.md [hunk 2]
```diff
@@ -18,10 +20,215 @@ One deck of [Acorn]({{site.baseurl}}/tree) cards.
...
+
+The *rotate* card will replace the left child of the *current* card. Since this rotation was done on a subtree, the *current* card will be placed where the *rotate* card used to be. The *current* card had no left child so the rotation is finished. The *flipped* card **25** is not the root of the tree and we need to Splay again.
+
+{% include figure image_path="/assets/images/splay14.png" alt="Tree splayed with Zig Zag operation." caption="Tree splayed with Zig Zag operation." %}
+
+Since the *flipped* card **25** is a child of the *root* card we will perform a *Zig* operation. Since the *flipped* card is the right child of the *root* card, we will perform a left rotation with the *flipped* card as our *current* card. The parent of our *current* card **19** will be our *rotate* card.
+
+The *rotate* card will replace the left child of the *current* card. The *current* card's left child that was replaced will be the right child of the *rotate* card. The *flipped* card **25** is now the root of the tree and we do not need to Splay again.
+
+{% include figure image_path="/assets/images/splay14.png" alt="Tree splayed with Zig operation." caption="Tree splayed with Zig operation." %}
```
Comment (mgoadric) [inline _pages/splaytree.md]: This last image is the same as the previous. Should it be splay15?

**Review by mgoadric — CHANGES_REQUESTED** [review]
Looks like a great setup! A few small corrections noted inline.

**Comment by qsebring3930** [comment]
Thanks for catching those, I corrected the image and added the learning goals.

**Comment by mgoadric** [comment]
This looks excellent, great work! In tabletopia I think you can select multiple cards at the same time and move them together. Rotating them together is another story...

**Comment by qsebring3930** [comment]
I'll have to look into that and try recording the video again, I know I've picked up more than one by accident before.

**Pull request merged**